public/app/features/dashboard-scene/panel-edit/LibraryVizPanelInfo.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, dateTimeFormat } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import { LibraryVizPanel } from '../scene/LibraryVizPanel';
+
+interface Props {
+  libraryPanel: LibraryVizPanel;
+}
+
+export const LibraryVizPanelInformation = ({ libraryPanel }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  const libraryPanelState = libraryPanel.useState();
+  const tz = libraryPanelState.$timeRange?.getTimeZone();
+  const meta = libraryPanelState._loadedPanel?.meta;
+  if (!meta) {
+    return null;
+  }
+
+  return (
+    <div className={styles.info}>
+      <div className={styles.libraryPanelInfo}>
+        {`Used on ${meta.connectedDashboards} `}
+        {meta.connectedDashboards === 1 ? 'dashboard' : 'dashboards'}
+      </div>
+      <div className={styles.libraryPanelInfo}>
+        {dateTimeFormat(meta.updated, { format: 'L', timeZone: tz })} by
+        {meta.updatedBy.avatarUrl && (
+          <img
+            width="22"
+            height="22"
```

Comment:
Probably we don't need to set the size if we're doing it already with CSS, do we? 
```suggestion
```